examples/russian.tl

```diff
@@ -0,0 +1,28 @@
+число результат = (5+2)
+              *5;
+
+напечатать результат;
+
+число результат = результат + 3;
+
+напечатать результат;
+
+строка имя = "Вера";
+
+напечатать имя;
+
+строка возраст = "20";
+число возраст = возраст + 10;
+напечатать возраст;
+
+если(результат == результат)
+  напечатать результат;
+конец
+
+асинхронн число x = 10;
+асинхронн напечатать x;
+
+напечатать "прибавить к x: ";
+
+читатьСтроку к число x;
+напечатать x;
```

Comment:
Awesome! Tested and is fully working right now! I've updated the `IsWord()` function.